core/src/util/mod.rs

```diff
@@ -1,6 +1,15 @@
 mod decimal128;
+use bson::{doc, Document};
 pub use decimal128::Decimal128Plus;
 
+// Replaces SQL wildcard characters with associated regex
```

Comment:
This was moved because it's also needed in fields.rs, though the main implementation will be in another patch, I don't feel like undoing this :P